### PR TITLE
update index.html and styles.css

### PR DESCRIPTION
**Pull request by Ilyaobr**

### CODE REVIEW

**Review by velmyk — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -97,11 +97,13 @@ <h1 class="title_hiring"> We're hiring </h1>
             <li class="company-description">Foundation is a website template
```
Comment (Nazar0n) [inline index.html]: I think tag `li` is better use with `ol` or `ul`, but not only 1 of there

File: index.html [hunk 2]
```diff
@@ -97,11 +97,13 @@ <h1 class="title_hiring"> We're hiring </h1>
...
                 for<br>startups and small teams. It helps to<br>build a website in
```
Comment (Nazar0n) [inline index.html]: Maybe use flex without tags `br`

Comment (velmyk) [reply index.html]: Just skip `<br>` and let the text flow as it is.
 With correct container width and font size your solution should look the same way as design do

File: index.html [hunk 3]
```diff
@@ -94,9 +96,11 @@ <h1 class="title_hiring"> We're hiring </h1>
                     <img class="logo" src="./images/picture-logo.png" width="60px" height="60px">
                 </div>
             </div>
-            <li class="company-description">Foundation is a website template
-                for<br>startups and small teams. It helps to<br>build a website in
-                no time.</li>
+            <div class="footer-column-for-description">
```
Comment (velmyk) [inline index.html]: Why do you need `div` and `span` around the text?

Comment (Ilyaobr) [reply index.html]: I tried to do without the 'div' but I couldn't indent like in the design

Comment (Nazar0n) [reply index.html]: Do you need `span?` there

Comment (Ilyaobr) [reply index.html]: there was 'li' I replaced it with 'span'. I don't know what else to replace it with

**Pull request merged**